### ISSUE TITLE
KotlinGenerator creates incorrect @Generated annotation

### ISSUE THREAD

**Issue by zarebski-m** [jOOQ/jOOQ#10771]
### Expected behavior
When using KotlinGenerator with option `generatedAnnotation = true` model should be generated correctly.

### Actual behavior
Model does not compile because the `@Generated` annotation uses Java syntax instead of Kotlin:

```
// what gets generated:
@Generated(
    value = {
        "https://www.jooq.org",
        "jOOQ version:3.14.0"
    },
    comments = "This class is generated by jOOQ"
)

// what should be generated:
@Generated(
    value = [
        "https://www.jooq.org",
        "jOOQ version:3.14.0"
    ],
    comments = "This class is generated by jOOQ"
)

// what also could be generated:
@Generated(
    "https://www.jooq.org",
    "jOOQ version:3.14.0",
    comments = "This class is generated by jOOQ"
)
```

**Comment by lukaseder** [comment]
Thanks a lot for your report. I can confirm this, will fix immediately.

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.15.0. Backport scheduled for 3.14.1 (#10773)

**Comment by lukaseder** [comment]
Thanks again for your report, and for your heads-up regarding the possible syntaxes!
